Lib/test/test_types.py

```diff
@@ -1376,6 +1376,27 @@ def __hash__(self):
         view = self.mappingproxy(mapping)
         self.assertEqual(hash(view), hash(mapping))
 
+    def test_richcompare(self):
+        mp1 = self.mappingproxy({'a': 1})
+        mp1_2 = self.mappingproxy({'a': 1})
+        mp2 = self.mappingproxy({'a': 2})
+
+        self.assertEqual(mp1, mp1_2)
+        self.assertFalse(mp1 != mp1_2)
+        self.assertFalse(mp1 == mp2)
+        self.assertNotEqual(mp1, mp2)
```

Comment:
I would prefer to call directly the operator:

```suggestion
        self.assertTrue(mp1 == mp1_2)
        self.assertFalse(mp1 != mp1_2)
        self.assertFalse(mp1 == mp2)
        self.assertTrue(mp1 != mp2)
```